package/DESCRIPTION

```diff
@@ -1,50 +0,0 @@
-Package: tlg.catalog.pkg
-Title: Tables, Listings and Graphs Catalog
-Version: 0.1.0.9009
-Date: 2023-02-21
-Authors@R: c(
-    person("Joe", "Zhu", , "joe.zhu@roche.com", role = c("aut", "cre")),
```

Comment:
why this file is removed?

Reply:
miscommunication - it's going to be restored

Reply:
Sorry about that, I will revert this :)

Reply:
reverted to the package structure 👍🏽